frontend/src/views/CellGuide/components/CellGuideInfoSideBar/style.ts

```diff
@@ -0,0 +1,5 @@
+import styled from "@emotion/styled";
+
+export const MarkerGeneTableWrapper = styled.section`
+  margin-top: -40px;
```

Comment:
rather than parametrize MarkerGeneTable's top margin and complicating the component's interface, we just throw it in a wrapper and subtract from the margin top (which is originally 80px).